### DISCUSSION TITLE
[Bug?] Products added to subcategory aren't automatically added to the parent category

### DISCUSSION THREAD

**Discussion by marnauortega** [swellstores/community#371] [Help] [answered]
When we add a product in a subcategory, it seems to implicitly be added to its parent category, as this image shows:

![Captura de Pantalla 2023-09-12 a les 20 35 38](https://github.com/swellstores/community/assets/37710726/0bd297d7-37c5-41a6-92ba-80ddf4fdfdda)

But it doesn't really get added to the parent, as this other image shows (there's 3 products in the subcategories, bot only 2 in the parent category):

![Captura de Pantalla 2023-09-12 a les 20 35 55](https://github.com/swellstores/community/assets/37710726/24f3fb70-553e-4c95-ac59-03fac62e07fa)

Is this intended?

**Comment by logeshswell** [comment, marked as answer]
@marnauortega The product won't be added to the parent category by default. It can be added by ticking the parent category checkbox explicitly.

As you can see, Products (Grey) > Original Art (Black) in your case which signifies Black (included) and Grey (excluded).

We will make sure to design it better for easy understanding.

**Reply by logeshswell** [reply]
Also, I can see that the color code changes for the category selection / unselection don't seem to be working well for hierarchical categories and we will look into it as well. Thanks for sharing it :)

**Reply by marnauortega** [reply]
Great to hear. But one thing @logeshswell , you want to allow the possibility of adding a product to a subcategory but not to its parent category? Is there a use case for this? Wouldn't we always want to have all products in a subcategory also belong to its parent category?

**Reply by logeshswell** [reply]
> Is there a use case for this?

I don't remember one, but different merchants come up with different use cases and hence we want to support the product features with endless flexibility.

**Reply by marnauortega** [reply]
I totally agree, flexibility is great.
Maybe the best user experience could be that when choosing a subcategory, the parent category gets also automatically selected. But there's still the chance to untick it, to only add the product to the subcategory.

**Reply by ericingram** [reply]
Good idea! 👆
